CMakeLists.txt

```diff
@@ -111,54 +111,50 @@ if (${run_unittests} OR ${run_int_tests})
     add_subdirectory(tests)
 endif()
 
-if(${use_installed_dependencies})
-    # Set CMAKE_INSTALL_* if not defined
-    include(GNUInstallDirs)
+# Set CMAKE_INSTALL_* if not defined
+include(GNUInstallDirs)
 
-    if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
-        set(CMAKE_INSTALL_LIBDIR "lib")
-    endif()
+if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+    set(CMAKE_INSTALL_LIBDIR "lib")
+endif()
+
+# Install UMock C
+set(package_location "cmake")
+
+install(TARGETS umock_c EXPORT umock_cTargets
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR}/../bin
```

Comment:
This should probably be `${CMAKE_INSTALL_BINDIR}` since you're using GNUInstallDirs. Also I think the above "if-not-defined" for LIBDIR is not needed because you're using GNUInstallDirs.

https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html